app/services/doc_auth/acuant/requests/get_results_request.rb

```diff
@@ -0,0 +1,34 @@
+module DocAuth
+  module Acuant
+    module Requests
+      class GetResultsRequest < DocAuth::Acuant::Request
+        attr_reader :instance_id
+
+        def initialize(config:, instance_id:)
+          super(config: config)
+          @instance_id = instance_id
+        end
+
+        def path
+          "/AssureIDService/Document/#{instance_id}"
+        end
+
+        def headers
+          super().merge 'Content-Type' => 'application/json'
```

Comment:
this is in a lot of places!

```suggestion
          super.merge('Content-Type' => 'application/json')
```